gruntfile.js

```diff
@@ -12,6 +12,10 @@ module.exports = function (grunt) {
       ],
       template: 'spec/support/test-runner/SpecRunner.html',
       outfile: 'spec/support/test-runner/WebRunner.html',
+      failFast: true,
```

Comment:
These don't work. run `npm run spec` and the options are not applied to the web runner. See the comment below on boot.js